dialogue-core/src/main/java/com/palantir/dialogue/core/PinUntilErrorNodeSelectionStrategyChannel.java

```diff
@@ -117,6 +117,7 @@ static PinUntilErrorNodeSelectionStrategyChannel of(
             return new PinUntilErrorNodeSelectionStrategyChannel(shuffling, initialPin, metrics, channelName);
         } else if (strategy == DialogueNodeSelectionStrategy.PIN_UNTIL_ERROR_WITHOUT_RESHUFFLE) {
             NodeList constant = new ConstantNodeList(initialShuffle);
+            log.info("Created PinUntilErrorNodeSelectionStrategyChannel", SafeArg.of("numHosts", constant.size()));
```

Comment:
I thought about logging in ChannelCache instead, but I didn't want to add extra logging volume on startup for everyone, so maybe this is a good enough of a compromise? this should be pretty much the one single client we care about?

Reply:
I think this may fill in the missing pieces of the puzzle: #1292

Reply:
Yeah